docs/docs-beta/docs/guides/external-systems/databases.md

```diff
@@ -1,4 +1,66 @@
 ---
 title: Connecting to databases
+description: How to configure resources to connect to databases
 sidebar_position: 10
----
\ No newline at end of file
+---
+
+In Dagster, *resources* are used to connect to databases by acting as a wrapper around database clients. The resource is registered along with connection details in the `Definitions` object, and can then be referenced from your asset definitions.
+
+## What you'll learn
+
+- How to connect to and query a local DuckDB database using the `DuckDBResource`
+- How to connect to different databases in different environments, such as development and production.
+- How to connect to a Snowflake database using the `SnowflakeResource`
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Familiarity with [Asset definitions](/concepts/assets)
+- Connection information for a Snowflake database
+
+</details>
+
+## Define a DuckDB resource and use it in an asset definition
+
+Here is an example of a DuckDB resource definition that's used to create two tables in the DuckDB database.
+
+<CodeExample filePath="guides/external-systems/resource-duckdb-example.py" language="python" title="DuckDB Resource Example" />
+
+To run the preceding example, first [install `dagster-duckdb`](https://dagster.io/integrations/dagster-duckdb) and then run:
+
+```shell
+$ dagster dev --python-file resource-duckdb-example.py
+```
+
+Materializing the assets from the Dagster UI will create the new DuckDB database at `/tmp/iris_dataset.duckdb`.
```

Comment:
Electing to remove instructions on how to run for this form of guide.

Will move `dagster-duckdb` as pre-req.
```suggestion
```